docs/generated/galasactl_streams_get.md

```diff
@@ -0,0 +1,33 @@
+## galasactl streams get
+
+Gets a list of test streams
+
+### Synopsis
+
+Get a list of test streams stored in the Galasa API server
```

Comment:
test streams aren't stored in the Galasa API server, this could be changed to something like "Get a list of test streams from the Galasa service"?